packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -69,6 +69,13 @@
           ".tokens"
         ]
       }
+    ],
+    "grammars": [
+      {
+        "language": "html",
+        "scopeName": "text.html.basic",
+        "path": "./syntaxes/html.tmLanguage.json"
```

Comment:
Is this scoped to just the above extensions or will it replace all html?

Reply:
Yes, it only applies to those file extensions.